bucket/sozi.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "21.12.06",
+    "description": "Zooming presentation editor and player.",
+    "homepage": "https://sozi.baierouge.fr/",
+    "license": "MPL-2.0",
+    "notes": "Install sozi extras media by running: \"scoop install inkscape-extension-sozi-extras\"",
```

Comment:
```suggestion
    "notes": "Install sozi extras media by running: \"scoop install inkscape-extension-sozi\"",
```

Reply:
I just wrote what it's said on inkscape UI, even the filename is longer `sozi-extras-media`. But ok